third_party/nvfuser/csrc/executor_utils.cpp

```diff
@@ -954,7 +954,6 @@ void dumpCompiledCodeToFile(
 }
 #endif
 
-#ifndef USE_ROCM
```

Comment:
Does this change have anything to do with magic zero? Or is it just cleanup? Can we remove all ROCM code now?

Reply:
No, this is just cleanup. I saw this in this file, so I removed it. I think we will not do any upstream bump before becoming a submodule, so I think it should be safe. cc @jjsjann123

Reply:
nvfuser is disabled in rocm now. https://github.com/pytorch/pytorch/pull/92182

It's safe for us to move forward with this cleanup

Reply:
It probably would still break the build though, since nvfuser is still build along with rocm.. We need to update that in upstream.

Reply:
@jjsjann123 Are you saying we should still keep the ROCM ifdefs? Or can we remove them now?

Reply:
Let's go ahead and remove them. I'll push changes to upstream to update ROCM build.